config/samples/logstash/logstash.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: elasticsearch-sample
```

Comment:
The purpose of this Elasticsearch resource is not clear.

Reply:
Done